### ISSUE TITLE
A problem in the file in genome build 37 downloaded using Ensembl Biomart is preventing to update database genes correctly

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#3327]
The downloaded file is incomplete and inconsistent. meaning of if you download it twice you get two different files, and that is causing the error in #3300

**Comment by leahkemp** [comment]
Hello! Thanks, I bumped into this issue too. Just a note in case someone else comes across the same thing as me. I had loaded genes with:

```bash
scout -db dhb-database-2 update genes --api-key SECRET_API_KEY --build 37
```

Because of this issue, this resulted in a reduced number of GRCh37 genes in scout.

I then loaded the PanelApp with:

```bash
scout -db dhb-database-2 load panel --panel-app --institute WDHB
```

Because of the reduced number of genes, it was skipping some genes in these panels so I'd end up with panels with missing genes.

After utilising the [fix_update_genes](https://github.com/Clinical-Genomics/scout/tree/fix_update_genes) branch, my gene list for GRCh37 seems to be fixed, I'm getting the same number of genes in my local scout instance as ya'll (https://github.com/Clinical-Genomics/scout/pull/3396#issuecomment-1152288441).

In order to fix my PanelApp gene panels, I needed to manually remove each panel with:

```bash
# specifying each panel id in the database
scout -db dhb-database-2 delete panel --panel-id 5763f2ea8f620350a1996048
```

And then re-run:

```bash
scout -db dhb-database-2 load panel --panel-app --institute WDHB
```

To get my PanelApp panels fixed.

Simply re-running the `scout -db dhb-database-2 load panel --panel-app --institute WDHB` command before manually removing each panel would cause the following error:

```bash
scout.exceptions.database.IntegrityError: Panel 391 with version 1.33 already exist in database
```

Scout seems to see the panel already in scout (even if it is not complete) and therefore not re-pull and update the panels

It's all fixed for me now, but for others, just a note to remove your panels before re-running the `scout load panel --panel-app` command. Also is there a way to remove all the PanelApp panels in scout in one command? Or perhaps more straightforward, forcing scout to re-load the PanelApp data? Apologies if I'm missing a command that already exists to do this :sweat_smile:

**Comment by northwestwitch** [comment]
Hi Leah, glad to hear that you managed to fix both genes and panel app collections! 😅 
Don't know if you decided to update genes because of the [PR fix](#3396) or if you found it after the mess was done, but we are now going to merge it so it'll be fixed once and for all.

Regarding the panels you got exactly the same problem we experienced during our last database upgrade, only we noticed the lack of many genes when we updated the OMIM panel instead. 

**At the moment there is no command to remove specific gene panels versions from the database**, but we must think about a solution, because we are basically dependent on external resources that we assume would work but errors happened and could happen again in the future . Perhaps the easiest solution might be adding a force load panel (or update panel) like --f or something, that when is present will force the panel to update maintaining the same version?

Let's have a new dedicated issue to this! 

Thanks for explaining your fix, might be useful for others that find themselves in the same situation!
